src/query/functions/binary/unless.go

```diff
@@ -44,10 +44,7 @@ func makeUnlessBlock(
 	}
 
 	lSeriesMetas := lIter.SeriesMeta()
-	lMeta, lSeriesMetas = removeNameTags(lMeta, lSeriesMetas)
-
 	rSeriesMetas := rIter.SeriesMeta()
-	rMeta, rSeriesMetas = removeNameTags(rMeta, rSeriesMetas)
```

Comment:
Any way to cover this in `unless_test.go`?

Reply:
It should be covered since it now checks for `test.NewSeriesMeta` as expected rather than `test.NewSeriesMetaWithoutName`

Reply:
Yes, I did update